devices/flash-flexspi/fspi.h

```diff
@@ -92,10 +90,14 @@ extern int flexspi_deinit(flexspi_t *fspi);
 extern void flexspi_setFlashSize(flexspi_t *fspi, const size_t *flashSizes, size_t count);
 
 
-/* Update FlexSPI sequence lookup table */
+/* Update FlexSPI sequence lookup table (raw table) */
 extern void flexspi_lutUpdate(flexspi_t *fspi, u32 index, const u32 *lutTable, size_t count);
```

Comment:
is this function still needed?

Reply:
Yes it's used in HyperBus implementation of nor flash.